### PR TITLE
Update 002_Half-Screen.txt

### PR DESCRIPTION
**Pull request by shoc71**
- 2 new levels added

### CODE REVIEW

File: Keep On Going/assets/levels/002_Half-Screen.txt [hunk 1]
```diff
@@ -2622,3 +2622,356 @@ self.respawn_zones =  [
...
+
+self.respawn_zones =  [
+]
+
+
+==============
+
+No_Network(897, 299, 1)
+
+self.draw = [pygame.draw.rect(screen, (30, 144, 255), [102, 52, 903, 440])]
```
Comment (slysnivy) [inline Keep On Going/assets/levels/002_Half-Screen.txt:2797]: Hopefully this works

Comment (slysnivy) [reply Keep On Going/assets/levels/002_Half-Screen.txt:2797]: Putting the self.draw twice

**Review by slysnivy — APPROVED** [review]
Nothing major, seems good otherwise

File: Keep On Going/assets/levels/002_Half-Screen.txt (hunk 1 above)
Comment (shoc71) [reply Keep On Going/assets/levels/002_Half-Screen.txt:2797]: Yep, it works. I checked before putting up the PR

File: Keep On Going/assets/levels/002_Half-Screen.txt [hunk 2]
```diff
@@ -2622,3 +2622,356 @@ self.respawn_zones =  [
 ]
 
 ======================
+
+Family_fued(76, 322, 1)
```
[the remark this answers is not in the document]
Comment (shoc71) [reply Keep On Going/assets/levels/002_Half-Screen.txt:2626]: LMFAO, we can always change that

**Pull request merged**